content/intro-to-storybook/react/zh-CN/simple-component.md

```diff
@@ -24,7 +24,7 @@ commit: 'c07ce59'
 
 我们将从 `Task` 的基本实现开始，简单传入我们需要的 `属性-props` 以及需要对任务执行的两个 `on` 操作（在列表之间移动它）：
 
-```javascript
+```jsx
```

Comment:
@Woodii1998 one quick tip for you as you're updating the translations. For code snippets you can use the following:

````js
```js:title=src/components/Task.js
import React from 'react';

export default function Task({ task: { id, title, state }, onArchiveTask, onPinTask }) {
  return (
    <div className="list-item">
      <label htmlFor="title" aria-label={title}>
        <input type="text" value={title} readOnly={true} name="title" />
      </label>
    </div>
  );
}
```
````

And for diff snippets, you can the following:
````
```diff:title=.storybook/main.js
module.exports = {
- stories: [
-   '../src/**/*.stories.mdx',
-   '../src/**/*.stories.@(js|jsx|ts|tsx)'
- ],
+ stories: ['../src/components/**/*.stories.js'],
  staticDirs: ['../public'],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/preset-create-react-app',
    '@storybook/addon-interactions',
  ],
  framework: '@storybook/react',
  core: {
    builder: '@storybook/builder-webpack5',
  },
  features: {
    interactionsDebugger: true,
  },
};
```
````

This was a feature introduced a while ago by a fellow team member of mine to help our readers get some clarity on the file that required changes or was being created, removing the need to identify the files through code comments at the top of the snippet. With that in mind, if you wouldn't mind updating what you already have to use this pattern and as you continue to work on the translation, apply it as well, we'd appreciate it.

Let me know if you have any questions, more than glad to answer them.